event-streams/src/main/java/com/redhat/cloud/platform/playbook_dispatcher/RunEventTransform.java

```diff
@@ -143,6 +145,31 @@ Payload buildRunPayload(Struct input) {
         payload.setCreatedAt(input.getString("created_at"));
         payload.setUpdatedAt(input.getString("updated_at"));
 
+        // If the option fields do not exist, set them to empty strings
+        try {
+            payload.setName(input.getString("name"));
```

Comment:
Checking if an optional field is defined by letting the code throw an exception and then catching is a bad practice. Because of the overhead of exceptions (capturing stack trace, etc.) they should only be used for error cases, not as control structures. Instead, it would be better to use the get method, do a null check and/or cast the result.